cmd/peapod-to-fstree/main.go

```diff
@@ -87,36 +87,34 @@ func main() {
 		log.Printf("processing shard %d...\n", i)
 
 		var (
-			countLimit  int
-			ppd, fstr   common.Storage
-			storagesCfg = sc.Blobstor
+			countLimit int
+			ppd, fstr  common.Storage
+			storageCfg = sc.Blobstor
 		)
 
-		for _, storageCfg := range storagesCfg {
-			switch storageCfg.Type {
-			case fstree.Type:
-				countLimit = storageCfg.CombinedCountLimit
-
-				fstr = fstree.New(
-					fstree.WithPath(storageCfg.Path),
-					fstree.WithPerm(storageCfg.Perm),
-					fstree.WithDepth(storageCfg.Depth),
-					fstree.WithNoSync(*storageCfg.NoSync),
-					fstree.WithCombinedCountLimit(countLimit),
-					fstree.WithCombinedSizeLimit(int(storageCfg.CombinedSizeLimit)),
-					fstree.WithCombinedSizeThreshold(int(storageCfg.CombinedSizeThreshold)),
-					fstree.WithCombinedWriteInterval(storageCfg.FlushInterval),
-				)
-
-			case peapod.Type:
-				ppd = peapod.New(
-					storageCfg.Path,
-					storageCfg.Perm,
-					storageCfg.FlushInterval,
-				)
-			default:
-				return fmt.Errorf("invalid storage type: %s", storageCfg.Type)
-			}
+		switch storageCfg.Type {
+		case fstree.Type:
+			countLimit = storageCfg.CombinedCountLimit
+
+			fstr = fstree.New(
+				fstree.WithPath(storageCfg.Path),
+				fstree.WithPerm(storageCfg.Perm),
+				fstree.WithDepth(storageCfg.Depth),
+				fstree.WithNoSync(*storageCfg.NoSync),
+				fstree.WithCombinedCountLimit(countLimit),
+				fstree.WithCombinedSizeLimit(int(storageCfg.CombinedSizeLimit)),
+				fstree.WithCombinedSizeThreshold(int(storageCfg.CombinedSizeThreshold)),
+				fstree.WithCombinedWriteInterval(storageCfg.FlushInterval),
+			)
+
+		case peapod.Type:
+			ppd = peapod.New(
```

Comment:
The problem for migrator is that it can't work if it has a single storage type, it has nothing to do.